editoast/src/views/stdcm_search_environment.rs

```diff
@@ -28,6 +33,12 @@ crate::routes! {
     "/stdcm/search_environment" => {
         create,
         retrieve_latest,
+        "/list" => {
+            list,
+        },
+        "/{env_id}" => {
+            delete,
+        },
```

Comment:
```suggestion
        "/list" => list,
        "/{environment_id}" => delete,
```

more concise + consistency with the rest of the url path

Reply:
Sure thanks!